resources/conf/log4j2.properties

```diff
@@ -5,7 +5,7 @@ appenders = ENFORCER_CONSOLE, ENFORCER_LOGFILE, ENFORCER_ACCESS_LOG
 appender.ENFORCER_CONSOLE.type = Console
 appender.ENFORCER_CONSOLE.name = ENFORCER_CONSOLE
 appender.ENFORCER_CONSOLE.layout.type = PatternLayout
-appender.ENFORCER_CONSOLE.layout.pattern = [%d{DEFAULT}][%X{traceId}] %5p - {%c} - %m [%errorDetails]%n
+appender.ENFORCER_CONSOLE.layout.pattern = [%d{DEFAULT}][%X{traceId}] %5p - {%c} - %m [%errorDetails] %x%n
```

Comment:
can include stack details after the traceId

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2679/commits/356dd76f69e350e032432200814911fd84269ee1